From 4018bee9ed089cab7fd493aeab491adaaa0f613d Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Sat, 5 Jan 2013 10:32:08 +0800
Subject: [PATCH 03/37] ENGR00232888 Connectivity:Fix network preference issue

Refactor getPersistedNetworkPreference and optimize for
updating mNetworkPreference according to user's networkAttributes
setting from overlay config.xml when connectivityservice start.
This will fix wifi/3G/Ethernet coexist and switch wrongly issue.
Merged by google internally--https://android-review.googlesource.com/#/c/46494/
Android4.2 still need this patch.Later version will include this.

Change-Id: I67624a8cf6ecd9e2d0cd14e2f60854512e417e3f
Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 .../src/com/android/providers/settings/DatabaseHelper.java               | 1 +
 services/java/com/android/server/ConnectivityService.java                | 1 +
 2 files changed, 2 insertions(+)

diff --git a/packages/SettingsProvider/src/com/android/providers/settings/DatabaseHelper.java b/packages/SettingsProvider/src/com/android/providers/settings/DatabaseHelper.java
index 09c21f3..d3a0850 100644
--- a/packages/SettingsProvider/src/com/android/providers/settings/DatabaseHelper.java
+++ b/packages/SettingsProvider/src/com/android/providers/settings/DatabaseHelper.java
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2007 The Android Open Source Project
+ * Copyright (C) 2013 Freescale Semiconductor, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
diff --git a/services/java/com/android/server/ConnectivityService.java b/services/java/com/android/server/ConnectivityService.java
index baff661..15cd228 100644
--- a/services/java/com/android/server/ConnectivityService.java
+++ b/services/java/com/android/server/ConnectivityService.java
@@ -1,5 +1,6 @@
 /*
  * Copyright (C) 2008 The Android Open Source Project
+ * Copyright (C) 2013 Freescale Semiconductor, Inc.
  *
  * Licensed under the Apache License, Version 2.0 (the "License");
  * you may not use this file except in compliance with the License.
-- 
1.8.0

